drivers/ws2812.h

```diff
@@ -56,6 +56,14 @@
 #    define WS2812_TRST_US 280
 #endif
 
+#if defined(LED_MATRIX_LED_COUNT)
+#    define WS2812_LED_NUM LED_MATRIX_LED_COUNT
+#elif defined(RGB_MATRIX_LED_COUNT)
```

Comment:
```suggestion
#if defined(RGB_MATRIX_LED_COUNT)
```
LED Matrix is for single-colour LEDs, the WS2812 driver won't be useful there.

Reply:
Thanks - fixed!